lib/matplotlib/axes/_axes.py

```diff
@@ -7104,16 +7104,19 @@ def hist(self, x, bins=None, range=None, density=False, weights=None,
     def stairs(self, values, edges=None, *,
                orientation='vertical', baseline=0, fill=False, **kwargs):
         """
-        A stepwise constant function as a line with bounding edges
-        or a filled plot.
+        Draw a stepwise constant function as a line or a filled plot.
```

Comment:
I've removed "with bounding edges", because the summary should be a single line. IMHO bounding edges is a detail and not essential for the summary.